pkg/specgenutil/specgen.go

```diff
@@ -873,7 +873,7 @@ func makeHealthCheckFromCli(inCmd, interval string, retries uint, timeout, start
 	}
 
 	var concat string
-	if cmdArr[0] == "CMD" || cmdArr[0] == "none" { // this is for compat, we are already split properly for most compat cases
+	if cmdArr[0] == "CMD" || cmdArr[0] == "NONE" { // this is for compat, we are already split properly for most compat cases
```

Comment:
Maybe do a `strings.ToUpper()` on cmdArr[0] so we don't have to worry about case?